src/textual/layouts/vertical.py

```diff
@@ -23,7 +21,7 @@ def arrange(
         placements: list[WidgetPlacement] = []
         add_placement = placements.append
 
-        y = max_width = max_height = 0
```

Comment:
PyCharm was highlighting that this initialisation of `y` is useless since we're re-initialising it later in the same scope, so I removed it from there :-)